APE1.rst

```diff
@@ -139,6 +139,28 @@ An APE's status can
   rationales, so that new community members can understand why a decision was
   made.
 
+Modifications to an APE
+.......................
+
+**Addendum: July 22, 2020**
+
+An APE can be modified, with one person acting as "champion" for the
+APE update. Similar to the APE development procedure, the update
+champion should discuss their desired changes with the astropy
+community (e.g., through the astropy-dev mailing list).
+
+Following a discussion on astropy-dev, the APE update shall be
+submitted as a Pull Request. The submission of the Pull Request should
+be announced formally on the astropy-dev list before they are
+accepted. If not already listed, the update champion can add
+themselves to the author list for the APE being updated. When
+appropriate, major changes to an APE document are encouraged to be
+clearly labeled as an addendum.
+
+The Astropy Coordinating Committee is responsible for reviewing and
+ultimately accepting Pull Requests for changes to individual APEs,
```

Comment:
This process should not be different than from the normal APE process, refer back to it instead of repeating. e.g. the committee is not responsible for accepting a PR, it can very well be categorized to any of APE status, even outright rejected.

Reply:
Maybe can use the `:ref:` tags. I wonder if those get rendered properly by GitHub. (Why didn't we go with markdown in the first place? Hmm.)

Reply:
Agree to just refer to the existing review process. No need to get hung up on making an internal link, just say `The review process is the same as for a newly-proposed APE as described in the Review section above.`

Reply:
The exception here though is that APE updates wouldn't be merged as rejected, so the process is slightly different? If an update is rejected we just close the PR?

Reply:
I implemented @taldcroft's specific wording suggestion